api/services/ocean/commons/parameters/clusterRollId.yaml

```diff
@@ -1,4 +1,4 @@
-in: query
+in: path
```

Comment:
No need to change, it was valid update
http://help.dev.spotinst.com/spotinst-api/ocean/ocean-cloud-api/ocean-for-ecs/roll-cluster/get/